EIPS/eip-5606.md

```diff
@@ -0,0 +1,117 @@
+---
+eip: 5606
+title: Multiverse NFTs
+description: A universal representation of multiple related NFTs as a single digital asset across various platforms
+author: Gaurang Torvekar (@gaurangtorvekar), Khemraj Adhawade (@akhemraj), Nikhil Asrani (@nikhilasrani)
+discussions-to: https://ethereum-magicians.org/t/eip-5606-multiverse-nfts-for-digital-asset-interoperability/10698
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-06
+requires: 721, 1155
+---
+
+## Abstract
+
+This specification defines a minimal interface to create a multiverse NFT standard for digital assets such as wearables and in-game items that, in turn, index the delegate NFTs on each platform where this asset exists. These platforms could be metaverses, play-to-earn games or NFT marketplaces. This proposal depends on and extends [EIP-721](./eip-721.md) and [EIP-1155](./eip-1155.md).
+
+The ERC specification defines an interface, which acts as an abstract layer on top of existing NFTs that we call a ‘Multiverse NFT’. A multiverse NFT indexes a digital asset and tracks its various incarnations across different metaverse platforms, games and such. We call the individual representations of this digital asset on the various platforms ‘delegate NFTs’. The standard also allows for the ‘bundling’ and ‘unbundling’ of these delegate NFTs within the multiverse NFT so holders can trade them individually or as a bundle.
```

Comment:
There's a lot of redundancy here. Try to eliminate 33-50% of the words.

Reply:
@Pandapip1 Done!